lib/patch_hcl.go

```diff
@@ -4,11 +4,11 @@ import (
 	"fmt"
 )
 
-func PatchSliceOfMaps(m map[string]interface{}, skip []string) map[string]interface{} {
-	return patchValue("", m, skip).(map[string]interface{})
+func PatchSliceOfMaps(m map[string]interface{}, skip []string, skipTree []string) map[string]interface{} {
```

Comment:
skipTree is awesome - I'm sure I've tried adding this before but for some reason didn't - perhaps I figured out it didn't actually solve the problem properly or something. Glad it helps here though.